### DISCUSSION TITLE
如何在同一頁面上有另一個 v5 編輯器實例？

### DISCUSSION THREAD

**Discussion by queirozmichel** [wangeditor-team/wangEditor#4970] [Q&A] [answered]
嘿！
我搜索了文档，但找不到任何关于在同一页面上创建多个 v5 编辑器实例的信息。
谁能帮我提供一个代码片段来显示如何正确执行此操作？

谢谢！

**Comment by wangfupeng1988** [comment, marked as answer]
Refer this demo https://www.wangeditor.com/demo/multi-editor.html?lang=en
